deploy/html/editform/editform_bone_relational.html

```diff
@@ -25,9 +27,14 @@
                 -
             </option>
         {% endif %}
-        <option value="{{ entry["key"] }}" {{ "selected" if boneValue and entry["key"] == boneValue["dest"]["key"] }}>
-            {{ entry["name"] }}
-        </option>
+            <option value="{{ entry["key"] }}"
+                    {% if boneParams.multiple %}
+                        {{ "selected" if entry["key"] in boneValue|map(attribute='dest.key') }}
+                    {% else %}
+                        {{ "selected" if boneValue and boneValue['dest'] and entry["key"] == boneValue["dest"]["key"] }}
+                    {% endif %}>
+                {{ entry["name"] }}
+            </option>
```

Comment:
Can you please add minuses to reduce the amount of whitespace:

```suggestion
            <option value="{{ entry["key"] }}"
                    {%- if boneParams.multiple -%}
                        {{ "selected" if entry["key"] in boneValue|map(attribute='dest.key') }}
                    {%- else -%}
                        {{ "selected" if boneValue and boneValue['dest'] and entry["key"] == boneValue["dest"]["key"] }}
                    {%- endif -%}>
                {{ entry["name"] }}
            </option>
```